File: javaScript/FloydsTriangle.js
function FloydsTriangle(rows) {
    let num = 1;
    for (let i = 0; i < rows; i++) {
        let row = '';
        for (let j = 0; j <= i; j++) {
            row += num + ' ';
            num++;
        }
        console.log(row);
    }
}

printFloydsTriangle(5);

File: java/FloydsTriangle.java
public class FloydsTriangle{
    public static void printFloydsTriangle(int rows) {
        int num = 1;
        for (int i = 0; i < rows; i++) {
            for (int j = 0; j <= i; j++) {
                System.out.print(num + " ");
                num++;
            }
            System.out.println();
        }
    }

    public static void main(String[] args) {
        FloydsTriangle(5);
    }
}